test/error.js

```diff
@@ -14,7 +13,7 @@ test('test error', function (t) {
     pull.values(['x', 'y', 'z']),
     pull.through(null, function (err) {
       if(_err) {
-        t.strictEqual(err, _err);
```

Comment:
I also made this change. I think that `err` and `_err` are identical objects, but they were failing because `{} !== {}`. The actual error is stored in `.error`, so I've changed this test to compare that property instead.